test/webpack.tests.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+
+const testsContext = require.context('.', true, /\-test\.js$/);
```

Comment:
Webpack stuff.
This way we have only one entry point instead of separate entry points for each test file. See https://github.com/webpack/karma-webpack#alternative-usage

Though if I think about it - for small modules we should keep it the default way. I switched to this alternative method on our main project because running tests (first run) on hundreds of test-files was extremely slow. Having one entry point fixed that in a very noticeable way.